go/testing/driver.go

```diff
@@ -192,7 +192,9 @@ func (c *Cluster) Verify(test *pf.TestSpec, testStep int, from, to *Clock) error
 	}
 
 	// Now feed documents into a combiner, filtering documents which are ACKs.
-	task, commons, _, err := c.Consumer.Catalog.GetTask(step.Collection.String())
+	// The waitForRevision parameter is 0 here because it's already guaranteed that the
+	// keyspace has observed the task because WaitForShardsToAssign has already been called.
```

Comment:
nit: Not _technically_ true. The `catalog` vs consumer service keyspace watches process independently, and it's conceivable that the consumer keyspace could be out ahead of the `catalog` keyspace.

I don't think this needs different behavior (for one, this is the same catalog keyspace that the consumer is using, and it's presumably already waited), but the comment should be clarified that we're assuming the race has already been won.